src/BlockID.h

```diff
@@ -486,13 +486,13 @@ enum
 	E_META_BREWING_STAND_FILLED_SLOT_XM_ZM = 4,
 
 	// E_BLOCK_BUTTON metas
-	E_BLOCK_BUTTON_YM      = 0,
-	E_BLOCK_BUTTON_XP      = 1,
-	E_BLOCK_BUTTON_XM      = 2,
-	E_BLOCK_BUTTON_ZP      = 3,
-	E_BLOCK_BUTTON_ZM      = 4,
-	E_BLOCK_BUTTON_YP      = 5,
-	E_BLOCK_BUTTON_PRESSED = 8,
+	E_META_BUTTON_YM      = 0,
+	E_META_BUTTON_XP      = 1,
+	E_META_BUTTON_XM      = 2,
+	E_META_BUTTON_ZP      = 3,
+	E_META_BUTTON_ZM      = 4,
+	E_META_BUTTON_YP      = 5,
+	E_META_BUTTON_PRESSED = 8,
```

Comment:
This is a breaking change for lua. So you either need to revert the change or add manual bindings for the old names.